hooks/useNodeJsContributorsApi.tsx

```diff
@@ -0,0 +1,181 @@
+import useSWR from 'swr';
+import { NodeJSContributorsApiKey } from '../constants/swr';
+
+const LIMIT_CONTRIBUTORS = 5;
+const CONTRIBUTORS_API_URI = `https://api.github.com/repos/nodejs/node/contributors?per_page=${LIMIT_CONTRIBUTORS}`;
+
+export type ParsedLink = {
+  url: string;
+  page: number;
+};
+
+export type ContributorApiResponse = {
```

Comment:
We don't need a type that covers the full JSON response, we can simply just leave what we want.

It is indeed a partial type that wouldn't reflect 100% on the extent of the object, but I rather doing that than keeping a bunch of types that have no value for me.